pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -730,6 +731,10 @@ func (m *kubeGenericRuntimeManager) doBackOff(pod *v1.Pod, container *v1.Contain
 		err := fmt.Errorf("Back-off %s restarting failed container=%s pod=%s", backOff.Get(key), container.Name, format.Pod(pod))
 		glog.Infof("%s", err.Error())
 		return true, err.Error(), kubecontainer.ErrCrashLoopBackOff
+		maxRestartTimes, err := strconv.Atoi(pod.ObjectMeta.Annotations["maxRestartTimes"])
```

Comment:
The code you added will never be exec'd because there is a `return` ahead of it...

Reply:
Oh, That's a merge mistake, I will fix it as soon as possible.

Reply:
Sorry for such stupid mistake, updated now.

Reply:
:)